Create delivery SubIdentity-milestone_3.md

a-oertel opened the pull request:
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/877

dsm-w3f commented:
@a-oertel and @dobschal thank you for the milestone submission and my apologies for the late response. Please see [the evaluation document](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/SubIdentity_3_dsm-w3f.md) and provide proper answers and fixes. After that, let me know to continue with the evaluation.

a-oertel commented:
Hey @dsm-w3f ,
thanks for your evaluation. We changed the delivery according to your findings.
The delivery is complete from our point of view.

**Here is the change log:**
* Fixed nginx config so that external links to the Identity Tab work. E.g. https://subidentity.tdsoftware.de/chain/westend/identity/5DJNbshySebwXdM43ZjfX2UcM8aDFoWkMbCeoskjCNumZEvH
* Fixed the Send Tokens button to send the correct amount
* Added indexed data for Kusama and Westend too
* Updated Tutorial to contain section about plugins on indexed chains
* Updated and added more unit tests
* Added more inline comments for better documentation
* Added e2e selenium tests for the frontend
* Updated the readme files to contain a "Get Started" section with step-by-step guide

Note: When opening the web app again, caching issues might occur. Please reload so that the latest version is loaded.

dsm-w3f commented:
Thank you @dobschal  and @a-oertel for the improvement. The milestone is approved and [the evaluation document](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/SubIdentity_3_dsm-w3f.md) was updated. Good work!
